code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -70,7 +70,7 @@
 		icon_state = "telebaton_1"
 		item_state = "telebaton_1"
 		w_class = 3
-		force = 50//Super Robust - Probably shouldn't give it to anyone who isn't staff unless you want BEATINGS
+		force = 40//Super Robust - Probably shouldn't give it to anyone who isn't staff unless you want BEATINGS
```

Comment:
This is still not acceptable. We want the baton to be something more like /tg/'s:
https://github.com/tgstation/tgstation/blob/e5e0459e334f2c71caf64f111cdd425f2991a89d/code/game/objects/items/melee/misc.dm

Reply:
It would be utterly worthless at 10 damage though.

Personally I don't give a shit so sure.